validate_release.py

```diff
@@ -63,30 +63,39 @@ def validate_installer(args, expectfile, unitfile):
                          '-bios', args.bios])
     cp.check_returncode()
 
-    print(">>> Installing boot canary into installer-target.mg")
-    cp = subprocess.run(['sudo', './install-canary.sh', 'installer-target.img'])
-    cp.check_returncode()
+    if not alternative_check:
```

Comment:
This is a bit messy but I think fine for now, we'll see if we get more of these to consolidate.

Reply:
ok